source/Node.js

```diff
@@ -177,12 +177,38 @@ function findNextTextOrNotEditable (root, node, options){
     return w.nextNONode(notEditable)
 }
 
+function commentHighlightId(node){
+    return node && node.getAttribute && node.getAttribute('data-comment-id')
+}
+
+function preventCommentHighlightMerge(n1, n2) {
+    var n1HighlightId = commentHighlightId(n1)
+    var n2HighlightId = commentHighlightId(n2)
+    // if neither are highlights, don't prevent
+    if (!n1HighlightId && !n1HighlightId){
+        return false
+    }
+    // if exactly one is a highlight, prevent merge
+    else if (!n1HighlightId || !n1HighlightId){
```

Comment:
When you return from `if` block, you don't need to write `else if` so another `if` sufficient.

Reply:
yeah, just a stylistic thing